### ISSUE TITLE
Request: Thousands and decimal separators on cart.currency

### ISSUE THREAD

**Issue by nikitaourazbaev** [Shopify/liquid#1704]
It would be great to be able to access the current currency’s thousands and decimals separators on the `cart.currency` object as currently there is no reliable way of getting them (for an example, see [Dawn’s price range filter](https://github.com/Shopify/dawn/blob/main/snippets/facets.liquid#L306-L312)).

On a related note, it would be great to be able to access these on `shop.currency` as well, but that object seems to be more limited than `cart.currency` (for example, it does not have the `symbol`)

**Comment by cfxd** [comment]
I agree, this would be great. For now, I'm getting the separator this way. It's not ideal and it's definitely gross, but it works for my purporses:

```
{%- capture money_separator -%}{{ 10000 | money_without_currency | replace: '1', '' | replace: '0', '' | replace: '0', '' | replace: '0', '' | replace: '0', '' }}{%- endcapture -%}
```
